packages/@storefront/flux-capacitor/test/unit/core/requests/utils.ts

```diff
@@ -55,21 +59,22 @@ suite('requests helpers', ({ expect, stub, spy }) => {
     it('should build out request', () => {
       stub(Selectors, 'config').returns({});
 
-      expect(RequestHelpers.search(<any>{})).to.eql({
+      expect(RequestHelpers.search(state)).to.eql({
         pageSize: remainingRecords,
         area,
         fields,
         query,
         collection,
         refinements,
+        sessionId,
         skip: originalSkip
       });
     });
 
     it('should decrease page size to prevent exceeding MAX_RECORDS', () => {
       stub(Selectors, 'config').returns({ search: {} });
 
-      const { pageSize, skip } = RequestHelpers.search(<any>{});
+      const { pageSize, skip } = RequestHelpers.search(state);
```

Comment:
This doesn't appear to be related to the update. Do we need to adjust this line?

Reply:
You're right it's not related, was required before my update to the way I was updating the test suite. Reverting these changes.